src/sentry/lang/javascript/errorlocale.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import, print_function
+
+import six
+import os
+import io
+import re
+
+LOCALES_DIR = 'src/sentry/data/error-locale'
```

Comment:
can we do `os.path.join(os.path.dirname(__file__), '../../data/error-locale'` instead? Current working directory is unrealiable. You can see with other code how we do it.